src/OpenTelemetry.Exporter.Zipkin/Implementation/ZipkinSpan.cs

```diff
@@ -112,7 +112,10 @@ public void Write(Utf8JsonWriter writer)
 
             writer.WriteString("id", this.Id);
 
-            writer.WriteString("kind", this.Kind);
+            if (this.Kind != null)
```

Comment:
The existing span exporter defaults to CLIENT, so this null situation never occurs.